### ISSUE TITLE
[jsdoc/require-description-complete-sentence] Inline JSDoc tags at the start of a sentence cause lint to fail

### ISSUE THREAD

**Issue by jnbooth** [gajus/eslint-plugin-jsdoc#1150]
This is similar to #705, but regarding capitalization at the start of a sentence rather than punctuation at the end of it.

## Expected behavior

If a JSDoc comment starts with an inline `{@link}` tag, jsdoc/require-description-complete-sentence should not fail as long as the text starts with a capital letter and ends with a period.

## Actual behavior

The above code sample fails jsdoc/require-description-complete-sentence because the linting rule thinks the description starts with `{`, which is not a capital letter.

## ESLint Config

```json
{
  "plugins": ["jsdoc"],
  "rules": {
   "jsdoc/require-description-complete-sentence": ["error"]
  }
}
```

## ESLint sample

```ts
/** @param options {@link RequestOptions} specifying path parameters and query parameters. */
```

## Environment

- Node version: 18.16.0
- ESLint version 8.18.0
- `eslint-plugin-jsdoc` version: 46.2.6

**Comment by brettz9** [comment]
I think we should just blanket permit inline tags at the beginning of a sentence. Besides the link possibly being a URL, it may be a variable name or such and not call for capitalization, so I think we should just allow any inline tags at the beginning.
